openadapt/crud.py

```diff
@@ -116,37 +177,57 @@ def insert_memory_stat(recording_timestamp, memory_usage_bytes, timestamp):
     _insert(memory_stat, MemoryStat, memory_stats)
 
 
-def get_memory_stats(recording_timestamp):
-    """
-    return memory stats for a given recording
-    """
-
+def get_memory_stats(recording_timestamp: int) -> None:
+    """Return memory stats for a given recording."""
     return (
-        db
-            .query(MemoryStat)
-            .filter(MemoryStat.recording_timestamp == recording_timestamp)
-            .order_by(MemoryStat.timestamp)
-            .all()
+        db.query(MemoryStat)
+        .filter(MemoryStat.recording_timestamp == recording_timestamp)
+        .order_by(MemoryStat.timestamp)
+        .all()
     )
 
 
-def insert_recording(recording_data):
+def insert_recording(recording_data: Recording) -> Recording:
+    """Insert the recording into to the db."""
     db_obj = Recording(**recording_data)
     db.add(db_obj)
     db.commit()
     db.refresh(db_obj)
     return db_obj
 
 
-def get_latest_recording():
+def get_latest_recording() -> Recording:
+    """Get the latest recording.
+
+    Returns:
+        Recording: The latest recording object.
+    """
     return db.query(Recording).order_by(sa.desc(Recording.timestamp)).limit(1).first()
 
 
-def get_recording(timestamp):
+def get_recording(timestamp: int) -> Recording:
+    """Get a recording by timestamp.
+
+    Args:
+        timestamp (int): The timestamp of the recording.
+
+    Returns:
+        Recording: The recording object.
+    """
     return db.query(Recording).filter(Recording.timestamp == timestamp).first()
 
 
-def _get(table, recording_timestamp):
+def _get(table: Any, recording_timestamp: int) -> list[Any]:
+    """Retrieve records from the database table based on the recording timestamp.
+
+    Args:
+        table (Any): The database table to query.
+        recording_timestamp (int): The recording timestamp to filter the records.
+
+    Returns:
+        list[Any]: A list of records retrieved from the database table,
```

Comment:
Please update or remove type in documentation

Reply:
resolved in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/4d079dc4023313ab796a1398cd95506a1885013c